go/shared.go

```diff
@@ -0,0 +1,21 @@
+package loyalty
+
+import "github.com/afitz0/customer-loyalty-workflow/status"
+
+type (
+	CustomerInfo struct {
+		CustomerId    string
```

Comment:
```suggestion
		CustomerID    string
```

Go traditionally capitalizes ID